### PR TITLE
[CL]: TestCreateFullRangePositionLocked

### PR DESCRIPTION
**Pull request by pysel**
Closes: #5156 

## What is the purpose of the change

Adds a direct unit test for `CreateFullRangePositionLocked`

Where is the change documented? 
N/A

### CODE REVIEW

File: x/concentrated-liquidity/position_test.go [hunk 1]
```diff
@@ -2071,3 +2071,92 @@ func (s *KeeperTestSuite) TestGetAllPositionIdsForPoolId() {
...
+		},
+	}
+
+	for _, test := range tests {
+		test := test
+		s.Run(test.name, func() {
+			// Init suite for each test.
+			s.SetupTest()
+			DefaultJoinTime := s.Ctx.BlockTime()
+			s.Ctx = s.Ctx.WithBlockTime(DefaultJoinTime)
```
Comment (p0mvn) [inline x/concentrated-liquidity/position_test.go:2131]: What's the point of this?

**Review by p0mvn — APPROVED** [review]
LGTM, nice work!

File: x/concentrated-liquidity/position_test.go (hunk 1 above)
Comment (pysel) [reply x/concentrated-liquidity/position_test.go:2132]: added some logic to test: https://github.com/osmosis-labs/osmosis/pull/5369/commits/084783ac887184496c4c6360eb5993f71c00d8ab.

basically, I use it to check position's `joinTime` field

**Review by AlpinYukseloglu — APPROVED** [review]
LGTM, thanks for adding this! Going to commit some cleanup changes (which might also be useful as future reference for similar PRs) and then I'll merge once CI passes

**Pull request merged**